chart/etcd-backup-restore/templates/etcd-statefulset.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 spec:
   updateStrategy:
     type: RollingUpdate
-  serviceName: {{ .Release.Name }}-etcd-client
+  serviceName: {{ .Release.Name }}-etcd-peer
   replicas: 1
```

Comment:
Additionally to my previous review, please make this field configurable via values.yaml, to support multinode etcd clusters.